Directory.Build.props

```diff
@@ -34,12 +34,12 @@
     <!-- Default to all packages generating a corresponding symbol package -->
     <IncludeSymbols>true</IncludeSymbols>
     <IsShippingPackage>false</IsShippingPackage>
-    <SdkTargetFramework>net9.0</SdkTargetFramework>
+    <SdkTargetFramework>net10.0</SdkTargetFramework>
     <ToolsetTargetFramework>$(SdkTargetFramework)</ToolsetTargetFramework>
     <VisualStudioServiceTargetFramework>net8.0</VisualStudioServiceTargetFramework>
 
     <!-- We used to have scenarios where the MSBuild host (VSMac) had an older .NET, but don't any more. -->
-    <ResolverTargetFramework>$(SdkTargetFramework)</ResolverTargetFramework>
+    <ResolverTargetFramework>net9.0</ResolverTargetFramework>
```

Comment:
Here is the build error message when set to net10.0.

`C:\Users\AzureTest\Desktop\109\sdk\.dotnet\sdk\10.0.100-alpha.1.24507.21\Microsoft.Common.CurrentVersion.targets(1889,5): error NU1702: ProjectReference 'C:\Users\AzureTest\Desktop\109\sdk\test\Microsoft.NET.TestFramework\Microsoft.NET.TestFramework.csproj' was resolved using '.NETFramework,Version=v4.7.2' instead of the project target framework '.NETCoreApp,Version=v9.0'. This project may not be fully compatible with your project.
    C:\Users\AzureTest\Desktop\109\sdk\.dotnet\sdk\10.0.100-alpha.1.24507.21\Microsoft.Common.CurrentVersion.targets(1889,5): error NU1702: ProjectReference 'C:\Users\AzureTest\Desktop\109\sdk\src\Cli\Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj' was resolved using '.NETFramework,Version=v4.7.2' instead of the project target framework '.NETCoreApp,Version=v9.0'. This project may not be fully compatible with your project.`

Reply:
What build did that trigger in? I tried locally and targeting net10.0 worked. The current builds are failing because the test framework isn't building because it doesn't have a net10.0 target (but projects that depend on it do have that). That means the nuget.config for the test projects never gets created. I pushed a change to switch back to net10.0 so we can see where this failure comes from.